### ISSUE TITLE
Indicator API not retrieving an indicator_value

### ISSUE THREAD

**Issue by Chris-Petty** [msupply-foundation/open-msupply#5422]
## What went wrong? 😲

In the indicator backend work I was reviewing and testing [this PR](https://github.com/msupply-foundation/open-msupply/pull/5347) and found for one updated indicator_value, it was no longer returned by the API. Cannot replicate this for any other indicator_value I try updating. I've included the OG db and pgdump of my OMS central where I was testing the API (Roxy was able to see the same problem).

## Expected behaviour 🤔

All updated values should continue to be returned from the API

## How to Reproduce 🔨

Steps to reproduce the behaviour:

OG Central:
[demo_data.zip](https://github.com/user-attachments/files/17742350/demo_data.zip)

PG Dump (OMS Central server):
[dump-omsupply-database-202411141615.sql.zip](https://github.com/user-attachments/files/17742345/dump-omsupply-database-202411141615.sql.zip)

Setup: OG central server and OMS central server

1. I made an internal order on OG and added some indicator values 1 and comment c1
2. I moved the store to OMS
3. I gql queried and saw my values
![image](https://github.com/user-attachments/assets/e76d4746-11f4-41cf-9231-6fdc5ce189b3)
4. I use mutate to change the first value from 1 to 245
![image](https://github.com/user-attachments/assets/f992e815-28a0-4d5a-a693-ab046aba9e1e)
5. I reran the same gql query and the value is now null
![image](https://github.com/user-attachments/assets/fc1d7464-0d88-419c-9813-55bb582b65e1)
6. It synced back to OG fine and I can see it fine, so what gives?
![image](https://github.com/user-attachments/assets/68ab8740-c5c9-4c2f-8b8b-d9d38e192633)
7. In the omSupply DB it does exist if you query for it. The following query is approximately what the OMS backend does... I think. 

```sql
SELECT
  indicator_value.id,
  indicator_value.customer_name_link_id,
  indicator_value.store_id,
  indicator_value.period_id,
  indicator_value.indicator_line_id,
  indicator_value.indicator_column_id,
  indicator_value.value
FROM
  indicator_value
WHERE
  indicator_value.customer_name_link_id = '6F0EA445B35F479AA1020BD2D24B9755'
  AND indicator_value.store_id = 'D77F67339BF8400886D009178F4962E1'
  AND indicator_value.period_id = '3D0504DD24134617B3982122AB5A81DA';
```
8. Proceed to do this for other values, but it works fine. The updated values show in the gql responses. But not that first one!

## Your environment 🌱

- Open mSupply Version: 2.4.0 DEV
- Legacy mSupply Central Server Version: 7.18 DEV
- Database type: Postgres

**Comment by andreievg** [comment]
Probably due to pagination limit in loader

**Comment by Chris-Petty** [comment]
> Probably due to pagination limit in loader

Hmm the record was there before though, would it get bumped to the end somehow 🤔.

One thing that is interesting is that the order of the loader doesn't seem guaranteed? I guess they're all evaluated async and can vary in order?

**Comment by roxy-dao** [comment]
Cause it's not being sorted. Do you want it sorted?

**Comment by roxy-dao** [comment]
Closed in #5435

**Comment by Chris-Petty** [comment]
> Cause it's not being sorted. Do you want it sorted?

Nah, just interested in how the gql layer works
